streams/src/main/java/org/apache/kafka/streams/kstream/internals/SessionCacheFlushListener.java

```diff
@@ -24,11 +24,11 @@
 import org.apache.kafka.streams.state.internals.CacheFlushListener;
 
 class SessionCacheFlushListener<K, V> implements CacheFlushListener<Windowed<K>, V> {
-    private final InternalProcessorContext context;
-    private final ProcessorNode myNode;
+    private final InternalProcessorContext<Object, Object> context;
+    private final ProcessorNode<?, ?> myNode;
```

Comment:
You'll notice that sometimes, I use wildcard (`?`) generic parameter bounds and other times, `Object`. The reason is a little subtle. It's because the InternalProcessorContext has a method that _takes_ a ProcessorNode argument. We want to let (ourselves) pass in _any_ ProcessorNode, not just an exactly `<Object, Object>` one. But for all the ProcessorContext references, _we_ pass the context in, and we have to do it untyped anyway, so the type bound doesn't matter. It results in less mind-bending code for users to declare types using the Object,Object bounds, and it makes no difference to them, as far as the calls to `forward`, since the parameterized types are only upper bounds anyway.

Punchline, it doesn't _really_ matter, but there are considerations that led me to use `?` in some cases and `Object` in others.